ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/viewcreation/valueobjects/ViewName.java

```diff
@@ -0,0 +1,53 @@
+package be.vlaanderen.informatievlaanderen.ldes.server.domain.viewcreation.valueobjects;
+
+import java.util.Objects;
+
+public class ViewName {
+
+	private final String name;
+	private final String collectionName;
+
+	public ViewName(String collectionName, String name) {
+		this.name = name;
+		this.collectionName = collectionName;
+	}
+
+	public ViewName withCollectionName(String collectionName) {
+		return new ViewName(collectionName, name);
+	}
+
+	public static ViewName fromString(String viewName) {
+		if (!viewName.contains("/")) {
+			throw new IllegalArgumentException(
+					"Invalid full view name: %s. '/' char expected after collectionName.".formatted(viewName));
+		}
+
+		final String[] splitViewName = viewName.split("/");
+		return new ViewName(splitViewName[0], splitViewName[1]);
+	}
+
+	public String getCollectionName() {
+		return collectionName;
+	}
+
+	@Override
+	public String toString() {
+		return collectionName + "/" + name;
+	}
```

Comment:
Minor: Would be careful with overwriting toString.
Personally, I would add here a method getCompleteViewName() and leave toString.